docs/application/dotnet/guides/nui/animation.md

```diff
@@ -53,10 +53,10 @@ animation.AnimateBy(view2, "PositionY", 100);
 animation.Play();
 ```
 
-### KeyFrame Animation
-NUI provides support for animating between several different values for a View. This can be done by defining the `KeyFrames`.
+### KeyFrames Animation
+By defining the `KeyFrames` object, NUI supports animation between several different values for a View.
```

Comment:
Please make the View in code tag as follows:

By defining the `KeyFrames` object, NUI supports animation between several different values for a `View`.